packages/ember-metal/lib/main.js

```diff
@@ -223,8 +223,8 @@ Ember._Cache = Cache;
 
 Ember.generateGuid    = generateGuid;
 Ember.GUID_KEY        = GUID_KEY;
-Ember.create          = Object.create;
-Ember.keys            = Object.keys;
+Ember.create          = Ember.deprecateFunction('Ember.create is deprecated in-favour of Object.create', Object.create);
```

Comment:
Should be `Ember.deprecateFunc` I believe (I don't think `Ember.deprecateFunction` is a thing).